db/seeds/minimal.seeds.rb

```diff
@@ -21,5 +21,9 @@
     :groups => [administrators]
 )
 
+# First entry for financial transaction types
+financial_transaction_class = FinancialTransactionClass.create(:name => "Other")
+FinancialTransactionType.create(:name => "Foodcoop", :financial_transaction_class_id => financial_transaction_class.id)
```

Comment:
Is it required to have a financial transaction type? Else I'd leave it out in the seeds.
It would be nice to have a database seed showcasing more extensive use of financial transaction classes.